cypress/component/QueryForm.cy.tsx

```diff
@@ -0,0 +1,167 @@
+import QueryForm from '../../src/components/QueryForm';
+
+const props = {
+  availableNodes: [
+    {
+      NodeName: 'Some Node Name',
+      ApiURL: 'https://someurl/',
+    },
+    {
+      NodeName: 'Another Node Name',
+      ApiURL: 'http://anotherurl/',
+    },
+  ],
+  diagnosisOptions: [
+    {
+      Label: 'Some Diagnosis',
+      TermURL: 'https://someurl/',
+    },
+    {
+      Label: 'Another Diagnosis',
+      TermURL: 'http://anotherurl/',
+    },
+  ],
+  assessmentOptions: [
+    {
+      Label: 'Some Assessment',
+      TermURL: 'https://someurl/',
+    },
+    {
+      Label: 'Another Assessment',
+      TermURL: 'http://anotherurl/',
+    },
+  ],
+  selectedNode: [],
+  minAge: null,
+  maxAge: null,
+  sex: null,
+  diagnosis: null,
+  isControl: false,
+  minNumSessions: null,
+  setIsControl: () => {},
+  assessmentTool: null,
+  imagingModality: null,
+  updateCategoricalQueryParams: () => {},
+  updateContinuousQueryParams: () => {},
+  loading: false,
+  onSubmitQuery: () => {},
+};
+
+describe('QueryForm', () => {
+  it('Displays a set of fields for user input and a button for submitting query', () => {
+    cy.mount(
+      <QueryForm
+        availableNodes={props.availableNodes}
+        diagnosisOptions={props.diagnosisOptions}
+        assessmentOptions={props.assessmentOptions}
+        selectedNode={props.selectedNode}
+        minAge={props.minAge}
+        maxAge={props.maxAge}
+        sex={props.sex}
+        diagnosis={props.diagnosis}
+        isControl={props.isControl}
+        minNumSessions={props.minNumSessions}
+        setIsControl={props.setIsControl}
+        assessmentTool={props.assessmentTool}
+        imagingModality={props.imagingModality}
+        updateCategoricalQueryParams={props.updateCategoricalQueryParams}
+        updateContinuousQueryParams={props.updateContinuousQueryParams}
+        loading={props.loading}
+        onSubmitQuery={props.onSubmitQuery}
```

Comment:
This may be a misunderstanding, but to assert that these fields show up at all, why do you need to provide every prop? Are they all required / do not have defaults?

Reply:
Yea they're all required.